optuna/__init__.py

```diff
@@ -20,3 +22,37 @@
 from optuna.study import Study  # NOQA
 from optuna.trial import Trial  # NOQA
 from optuna.version import __version__  # NOQA
+from optuna.type_checking import TYPE_CHECKING  # NOQA
+
+
+if TYPE_CHECKING:
+    from optuna import dashboard  # NOQA
+else:
+    from typing import Any  # NOQA
```

Comment:
I think that `# NOQA` comment isn't required here since `Any` is used at line 55.
```suggestion
    from typing import Any
```